.github/workflows/main.yml

```diff
@@ -33,8 +34,20 @@ jobs:
         run: |
           nix-instantiate --json --eval --strict -E 'with builtins; attrNames (getFlake (toString ./.)).checks.${currentSystem}' | perl -pe 's|(.*)|checks=\1|' >>$GITHUB_OUTPUT
 
+      - name: Generate Version 🏷️
+        id: version
+        run: |
+          TAG_COUNT=$(git rev-list --tags --no-walk --count)                                                                 # Count all tags
```

Comment:
Not sure if we need that at all. If we ever delete a tag this will be off.

Reply:
True, but I don't see a better way here. The relevant code is already on main (#183). Not being able to delete tags is something I'd be fine with, honestly. If you have a better Idea, we can still change it though